pkg/kubelet/dockershim/docker_service.go

```diff
@@ -379,6 +386,27 @@ func (ds *dockerService) GenerateExpectedCgroupParent(cgroupParent string) (stri
 	return cgroupParent, nil
 }
 
+// checkVersionCompatibility verifies whether docker is in a compatible version.
+func (ds *dockerService) checkVersionCompatibility() error {
+	apiVersion, err := ds.getDockerAPIVersion()
+	if err != nil {
+		return err
+	}
+
+	minApiVersion, err := semver.Parse(fmt.Sprintf("%s.0", dockertools.MinimumDockerAPIVersion))
```

Comment:
Let's just declare a new the minimum docker apiversion in dockershim as "1.22.0"